main.go

```diff
@@ -68,68 +59,16 @@ func main() {
 		arrange.ForViper(v),
 		fx.Supply(logger),
 		fx.Supply(v),
+		provideAuth("authx.inbound"),
 		touchstone.Provide(),
 		touchhttp.Provide(),
-		basculemetrics.ProvideMetricsVec(),
-		basculechecks.ProvideMetricsVec(),
-		auth.ProvidePrimaryServerChain(apiBase),
 		fx.Provide(
-			backwardsCompatibleMetricFactory("prometheus"),
-			backwardsCompatibleUnmarshaller,
-			backwardsCompatibleLogger,
-			auth.ProfilesUnmarshaler{
-				ConfigKey:        "authx.inbound.profiles",
-				SupportedServers: []string{"primary"}}.Annotated(),
+			consts,
+			gokitLogger,
```

Comment:
What is this needed for? Can we get rid of it?

Reply:
we don't 🎉  so I removed this